iOS/ARShooterExample/ARShooterGame/ARShooter/LoginViewController/LoginViewController.swift

```diff
@@ -131,6 +159,32 @@ class LoginViewController: UIViewController {
         }
     }
     
+    // This function shows the GetConnection workflow
+    func getWebsocketConnection() {
+        let replyPromise = matchingEngine.registerAndFindCloudlet(devName: devName, appName: appName, appVers: appVers, carrierName: carrierName, authToken: authToken, gpsLocation: location!, uniqueIDType: uniqueIDType, uniqueID: uniqueID, cellID: cellID, tags: tags)
```

Comment:
This API is getting unwieldy huge with parameters...

Reply:
Should we make a default constructor (like in android)?

Reply:
Depends, Swift has optional and default value parameters. If it can scale up and down using language features, we're actually fine. Android/Java/Kotlin has NO default parameters with values, hence existence of the "builder" API to ... do default values.